### ISSUE TITLE
Misleading warning

### ISSUE THREAD

**Issue by emartynov** [NeoTech-Software/Android-Root-Coverage-Plugin#45]
Super thank you for the work under the plugin.

We have next project structure:

```
root 
   features 
      registration
      fit
      ...
  core
     network
     ...
  app
```

We are getting warning that projects `features` and `core` are not supported. However, they are not projects but just container folders.

**Comment by Rolf-Smit** [comment]
@emartynov this is something I will fix in the next release.

**Comment by Rolf-Smit** [comment]
In the next release instead of a warning this will likely become something like this:

> _Notice: skipping module `x` because Android Gradle Plugin was not detected_

Which is also not going to be logged on the WARNING level but on the INFO level.

**Comment by emartynov** [comment]
Rolf, it is still misleading since they are not modules

**Comment by Rolf-Smit** [comment]
From a Gradle perspective they are actually seen as modules (that is also why the warning is triggered for.those anyways), so it might be hard to differentiate.

But I will see what I can do.

**Comment by Rolf-Smit** [comment]
Fixed in #50, will be released in version 1.5.

**Comment by Rolf-Smit** [comment]
I've added a check which checks if a Gradle sub(project) is an empty container for other (sub)projects, like in your case. If that is the case it will not log a message. "empty container" means in this case that there is no `build.gradle` file in that folder.

This is released with version 1.5.0.
